xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
+	step again. See also question on live migration failure.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node has failed at the initial chef client run stage.
+       </para>
+      </question>
+      <answer>
+       <para>
+	An unsupported entry in the configuration file may prevent a service
+	from starting. This, in turn, causes the node to fail at the initial
+	chef client run stage. Checking the
+	<filename>/var/log/crowbar/crowbar_join/chef.*</filename> log files on
+	the node is a good starting point.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	I need to change &ostack-bare; configuration during the upgrade but I cannot access &crow;.
+       </para>
+      </question>
+      <answer>
+       <para>
+	&crow; &wi; is accessible only when upgrade is completed or
+	when it is postponed. Postponing the upgrade can be done only after
+	upgrading all &contrnode;s using the <command>crowbarctl upgrade nodes
+	postpone</command> command. You can then access &crow; and
+	even save some modification. It is then necessary to resume the upgrade
+	using the <command>crowbarctl upgrade nodes resume</command> command
+	before you can continue with the upgrade of rest of the nodes.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Failure occurred when evacuating routers.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check the <filename>/var/log/crowbar/node-upgrade.log</filename> file on
+	the node that performs the router evacuation (it should be mentioned in
+	the error message). The ID of the router that failed to migrate (or the
+	affected network port) is logged to
+	<filename>/var/log/crowbar/node-upgrade.log</filename>). Use the
+	&ostack-bare; CLI tools to check the state of the affected router and
+	its ports. Fix manually, if necessary. This can be done, for example,
+	by bringing the router or port up and down again. The following
+	commands can be useful for solving the issue: <command>openstack router
+	show <replaceable>ID</replaceable></command>, <command>openstack port
+	list --router <replaceable>ROUTER-ID</replaceable></command>,
+	<command>openstack port show
+	<replaceable>PORT-ID</replaceable></command>, <command>openstack port
+	set</command>. Resume the upgrade by running the failed upgrade step
+	again to continue with the router migration.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade status file is empty or corrupted. <command>crowbarctl upgrade status</command> reports invalid information.
+       </para>
+      </question>
+      <answer>
+       <para>
+	This can happen if the &admserv; runs out of disk space or &crow;
+	service is stopped abruptly during a status write operation. Contact
+	support for assistance on how to recover the status file.
```

Comment:
I'm not sure if this answer is helpful at all. I'm inclined to delete the paragraph completely. @rhafer @skazi0 ideas?

Reply:
@jsuchome we don't want to write full "how to make a status file by hand" guide, do we? The idea behind this hint was to make people aware that this file could be sometimes re-created if lost. If you think that this doesn't bring much benefit then just drop this sentence.

Reply:
> we don't want to write full "how to make a status file by hand" guide, do we? 

Yeah, that sounds too dangerous. People could get easily into unsupported scenario.

I'm honestly not sure. I guess when they find out that their upgrade is broken, they know to contact us anyway.

Reply:
Just include a "how to contact support" note. Seems obvious - but people want the information *there*

Reply:
If the solution is to contact support, then it's not a solution at all. In which case, I vote for removing this part altogether.

Reply:
I'd say drop it